endurance-page-cache.php

```diff
@@ -831,7 +831,7 @@ public function htaccess_contents_rewrites( $rules ) {
 			$additions = "<ifModule mod_headers.c>\n" . 'Header set X-Endurance-Cache-Level "' . $this->cache_level . '"' . "\n</ifModule>\n";
 
 			if ( false === strpos( dirname( __FILE__ ), 'public_html' ) ) {
-				$additions .= 'Options -Indexes ' . "\n" . '
+				$additions .= '# BEGIN ENDURANCE PAGE CACHE REWRITES - set Assets Only to disable' . "\n" . 'Options -Indexes ' . "\n" . '
```

Comment:
Should we move this up and wrap the comments around the `X-Endurance-Cache-Level` header portion too? Or what about the stuff from `htaccess_contents_expirations()`?